api/v1alpha2/virtualmachine_types.go

```diff
@@ -23,11 +23,24 @@ const (
 	// For more information please see VirtualMachineImage.Status.Ready.
 	VirtualMachineConditionImageReady = "VirtualMachineImageReady"
 
-	VirtualMachineConditionNetworkReady = "VirtualMachineNetworkReady"
+	// VirtualMachineConditionVMSetResourcePolicyReady indicates that a referenced
+	// VirtualMachineSetResourcePolicy is Ready.
+	VirtualMachineConditionVMSetResourcePolicyReady = "VirtualMachineConditionVMSetResourcePolicyReady"
 
+	// VirtualMachineConditionStorageReady indicates that the storage prerequisites for the VM are ready.
 	VirtualMachineConditionStorageReady = "VirtualMachineStorageReady"
 
+	// VirtualMachineConditionBootstrapReady indicates that the bootstrap prerequisites for the VM are ready.
 	VirtualMachineConditionBootstrapReady = "VirtualMachineBootstrapReady"
+
+	// VirtualMachineConditionNetworkReady indicates that the network prerequisites for the VM are ready.
+	VirtualMachineConditionNetworkReady = "VirtualMachineNetworkReady"
+
+	// VirtualMachineConditionPlacementReady indicates that placement, if required for the VM, was done and is ready.
```

Comment:
When placement is `not` required for this VM, do we skip this condition or this condition is set to ready regardless?